cypress/integration/racing.spec.js

```diff
@@ -0,0 +1,152 @@
+import { contentType } from "mime-types";
+import { MESSAGE } from "../../src/js/constant.js";
+
+describe("racing", () => {
+  beforeEach(() => {
+    cy.visit("http://127.0.0.1:5500/");
+  });
+
+  const arrayToString = (ary) => {
+    return ary.join(", ");
+  }
+
+  const inputCarName = (ary, isSuccess) => {
+    let str = arrayToString(ary);
+    cy.get("#carname>input").type(str);
+    cy.get("#carname>button").click();
+    if (isSuccess){
+      cy.get("#carname>button").should("be.disabled");
+      cy.get("#count_filed").should("be.exist");
+    }
+    else{
+      cy.get("#count_filed").should("not.exist");
+      cy.on('window:alert',(txt)=>{
+        expect(txt).to.contains(MESSAGE.CAR_NAME);
+      })
+    }
+  }
+
+  const proceedRacing = (time, names) => {
+    let distance;
+    cy.wait(time).get(".mr-2").find("div.car-player").then(($div)=>{
+      distance = new Array($div.length).fill(0);
+      for(let i = 0; i < $div.length; i++){
+        let nodes = $div[i].parentNode.childNodes;
+        nodes.forEach((x) => {
+           if (x.nodeName != "#text" && x.className.indexOf("forward") >= 0){
+             distance[i]++;
+           }
+        });
+      }
+    });
+    cy.get("#result_section").should("be.exist");
+    cy.get("#result_section").find("h2").then(($h2)=>{
+      let max = Math.max.apply(null, distance);
```

Comment:
spread operator 사용하시면 더 간결하게 표현가능해요 ㅎㅎ 그리고 distance는 배열이니까 복수인 distances로 표현하는 것이 조금 더 와닿을 것 같아요!
```suggestion
      let max = Math.max(...distance);
```

Reply:
단순히 배열에서 최대값을 구하는 방법으로 생각없이 사용하던 방식이었는데 이렇게 동작하는 코드였군요! 확실하게 짚고 넘어가야겠습니다!